config/env/development.js

```diff
@@ -0,0 +1,25 @@
+export default {
+  app: {
+    name: 'Cards for Humanity - Development'
+  },
+  facebook: {
+    clientID: process.env.FB_Dev_clientID,
+    clientSecret: process.env.FB_Dev_clientSecret,
+    callbackURL: process.env.FB_Dev_callbackURL
+  },
+  twitter: {
+    clientID: process.env.TW_Dev_clientID,
+    clientSecret: process.env.TW_Dev_clientSecret,
+    callbackURL: process.env.TW_Dev_callbackURL
+  },
+  github: {
+    clientID: 'APP_ID',
+    clientSecret: 'APP_SECRET',
+    callbackURL: 'http://localhost:3000/auth/github/callback'
+  },
+  google: {
+    clientID: process.env.Google_Dev_clientID,
+    clientSecret: process.env.Google_Dev_clientSecret,
+    callbackURL: process.env.Google_Dev_callbackURL
+  }
+};
```

Comment:
Put the constants in all caps. For instance, change 'process.env.TW_Dev_callbackURL' to 'process.env.TW_DEV_CALLBACK_URL'